src/modules/rover_differential/module.yaml

```diff
@@ -113,6 +100,21 @@ parameters:
         decimal: 2
         default: 0.5
 
+      RD_MAX_DECEL:
+        description:
+          short: Maximum deceleration
+          long: |
+            Maximum decelaration is used to limit the deceleration of the rover.
+            Set to -1 to disable, causing the rover to decelerate as fast as possible.
```

Comment:
Doesn't 0 also disable it?

Reply:
Yes, but isn't the convention to set a parameter to -1 to disable it?

Reply:
Set default to -1 to make it easier to understand.